README.md

```diff
@@ -69,35 +71,59 @@ Add this plugin like this:
 
 ## Example usage
 
-We create a subfeed for messages which describe us under our `root` feed
+Let's start by creating a **root metafeed** with `findOrCreate(cb)`, necessary
+for using this module. There can only be one _root_ metafeed, so even if you
+call `findOrCreate(cb)` many times, it will not create duplicates, it will just
+load the root metafeed.
+
 ```js
-sbot.metafeeds.findOrCreate((err, rootFeed) => {
-  console.log(rootFeed)
+sbot.metafeeds.findOrCreate((err, root) => {
+  // ...
+})
+```
+
+Now we create a subfeed for `about` messages under our `root` feed using
+`findOrCreate(parent, isFeed, details, cb)`. This will only create the subfeed
+if there is no existing subfeed that matches the criteria`isFeed`:
 
-  // find an existing feed
+```js
+sbot.metafeeds.findOrCreate((err, root) => {
+  // Find an existing feed
   const isFeed = (feed) => feed.feedpurpose === 'aboutMe'
-  // details for creating a sub-feed feed if it doesn't already exist
+  // Details for creating a subfeed if it doesn't already exist:
   const details = { feedpurpose: 'aboutMe', feedformat: 'classic' }
 
-  sbot.metafeeds.findOrCreate(rootFeed, isFeed, details, (err, aboutMeFeed) => {
-    console.log(aboutMeFeed)
-
+  sbot.metafeeds.findOrCreate(root, isFeed, details, (err, aboutMeFeed) => {
```

Comment:
I liked rootFeed, makes it clear that it's the same as aboutMeFeed

Not gonna die, just think it makes it clearer than root